frontend/lib/norent/letter-content.tsx

```diff
@@ -76,13 +100,15 @@ function friendlyUTCDate(date: GraphQLDate) {
   return friendlyDate(new Date(date), "UTC");
 }
 
-const PaymentDate: React.FC<{ paymentDate: GraphQLDate }> = (props) => (
-  <>{friendlyUTCDate(props.paymentDate)}</>
+const PaymentDate = componentizeHelper((props) =>
+  friendlyUTCDate(props.paymentDate)
 );
 
 const LetterHeading: React.FC<NorentLetterContentProps> = (props) => (
   <dl className="jf-letter-heading">
-    <dt>To</dt>
+    <dt>
+      <Trans description="before address in formal letter">To</Trans>
```

Comment:
I tried adding a "description" here and some other places to make it easier for translators to understand the context.

Reply:
Nice!

Reply:
Do these show up in CrowdIn too?

Reply:
Curious to see how these show up in CrowdIn!